src/exec.js

```diff
@@ -21,6 +23,9 @@ function execSync(cmd, opts) {
     silent: common.config.silent
   }, opts);
 
+  var maxBuffer = parseInt(opts.maxBuffer, 10) || DEFAULT_MAXBUFFER_SIZE;
```

Comment:
@pieterjandesmedt: Yeah, I meant that I don't think we need our API to support someone passing in an invalid value. 

But now that I think about it, this line will end up being useful for #244, so forget I ever said anything.